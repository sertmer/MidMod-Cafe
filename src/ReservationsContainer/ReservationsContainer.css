div {
  display: flex;
  flex-flow: row wrap;
}