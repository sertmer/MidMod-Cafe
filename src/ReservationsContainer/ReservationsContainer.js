import React from 'react'
import './ReservationsContainer.css'
import ReservationsCard from '../ReservationsCard/ReservationsCard'

const ReservationsContainer = ({ reservations, removeReservation }) => {
  let reservationCards = reservations.map(reservation => {
    return <ReservationsCard 
              key={reservation.id}
              removeReservation={removeReservation}
              {...reservation}/>
  })
  return (
    <div>
      {reservationCards}
    </div>
  )
}

export default ReservationsContainer