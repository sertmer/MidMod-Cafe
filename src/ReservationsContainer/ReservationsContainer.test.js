import React from 'react';
import { shallow } from 'enzyme';
import ReservationsContainer from './ReservationsContainer';

describe('ReservationsContainer', () => {
  let wrapper
  let mockReservations
  let mockRemoveReservation
  beforeEach(() => {
    mockReservations = [{
      "id": 1,
      "name": "Christie",
      "date": "12/29",
      "time": "7:00",
      "number": 12
  }]

    mockRemoveReservation = jest.fn
    wrapper = shallow(<ReservationsContainer 
      reservations={mockReservations}  
      removeReservation={mockRemoveReservation}
    />)
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})