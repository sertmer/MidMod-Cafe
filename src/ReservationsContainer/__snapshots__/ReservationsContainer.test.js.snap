// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationsContainer should match the snapshot 1`] = `
<div>
  <ReservationsCard
    date="12/29"
    id={1}
    key="1"
    name="Christie"
    number={12}
    removeReservation={[Function]}
    time="7:00"
  />
</div>
`;
